lib/src/main/java/io/github/taz03/jia/InstagramClient.java

```diff
@@ -49,20 +56,40 @@ public final class InstagramClient {
      * @param password The instagram password corresponding to the username provided
      */
     public InstagramClient(String username, String password) {
-        this(username, password, null);
+        this.username = username;
+        this.password = password;
+    }
```

Comment:
set a default userAgent and androidId in this case

Reply:
when user calls `login()`, they are assigned automatically and randomly